tests/js/spec/views/projectTeamSettings.spec.jsx

```diff
@@ -78,10 +78,13 @@ describe('ProjectTeamsSettings', function() {
       expect(mock).not.toHaveBeenCalled();
 
       // open modal
-      wrapper.find('Button').simulate('click');
```

Comment:
curious why this doesn't work

Reply:
it was also finding the "create a new team" button from the dropdown and i was getting `Method “simulate” is only meant to be run on a single node. 2 found instead.`